source/features/comment-to-draft-pr-button.tsx

```diff
@@ -0,0 +1,22 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import onNewComments from '../github-events/on-new-comments';
+
+function init(): void {
+	if (select.exists('#partial-discussion-header span[title="Status: Draft"]')) {
+		const commentButton = select('#partial-new-comment-form-actions > div > div.color-bg-secondary.ml-1 > button');
```

Comment:
Selectors should be shorter and smarter than this. The button has a `.btn-primary` class and `type=submit` that are way more specific and don't need almost any other specifiers. An example could be `#partial-new-comment-form-actions .btn-primary`

Reply:
Thank you both for the suggestions. I followed them all in the new commit + updated the screenshot.